README.md

```diff
@@ -69,6 +69,7 @@
 ### C++
 
 - [WPILib (Java/C++)](https://github.com/wpilibsuite/allwpilib) - The official WPILib repository, containing code for the Java and C++ libraries.
+- [barebonesFRC](https://github.com/Ewpratten/barebonesFRC) (still in development)
```

Comment:
Please add a description of what this is rather than just labeling it as in development. Also, follow the general style: needs a hyphen in between the link and the description.